### PR TITLE
Iteration 3 mb

### PR DESCRIPTION
**Pull request by Maxwell-Baird**
Created two new files call game_teams and game_teams_stats
created the test for the two files
fixed some hardcoding

### CODE REVIEW

File: lib/game_stats.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class GameStats
   attr_reader :games
 
   def initialize(file_path, object)
-    @games = csv_data('./data/games_truncated.csv', Game)
+    @games = csv_data(file_path, object)
```
Comment (alex-latham) [inline lib/game_stats.rb:9]: Good job making this dynamic

File: test/test_game_teams_stats.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require_relative 'test_helper'
+require './lib/data_loadable'
+require './lib/game_teams'
+require './lib/game_teams_stats'
+
+class GameteamsStatsTest < Minitest::Test
+
+  def setup
+    @game_Team_stats = GameTeamStats.new("./data/game_teams_truncated.csv", GameTeams)
```
Comment (tylertomlinson) [inline test/test_game_teams_stats.rb]: maybe Team should be capitalized in the IV

File: test/test_game_teams_stats.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  def test_it_exists
+    assert_instance_of GameTeamStats, @game_Team_stats
```
Comment (tylertomlinson) [inline test/test_game_teams_stats.rb:13]: here too

File: lib/game_stats.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@ class GameStats
   attr_reader :games
 
   def initialize(file_path, object)
-    @games = csv_data('./data/games_truncated.csv', Game)
```
Comment (tylertomlinson) [inline lib/game_stats.rb:9]: Nice. Hard coding for n00bs

File: test/test_game_teams.rb [hunk 5]
```diff
@@ -0,0 +1,37 @@
+require_relative 'test_helper'
+require './lib/game_teams'
+
+class TeamTest < Minitest::Test
```
Comment (tylertomlinson) [inline test/test_game_teams.rb]: GameTeam

**Pull request merged**